### ISSUE TITLE
Increase the number of bits of the CT rangeproofs

### ISSUE THREAD

**Issue by stevenroose** [ElementsProject/elements#792]
Since we're reducing the fee rates, we might as well make this change that's been talked about for a while.

**Comment by instagibbs** [comment]
right, in lieu of bulletproofs(long deferred) this may be preferable.

**Comment by allenpiscitello** [comment]
We should consider waiting on doing this until after the lower fee has been deployed in Liquid.

**Comment by stevenroose** [comment]
This has been implemented.
